apps/aptos/views/Ifos/hooks/v3/useGetPublicIfoData.ts

```diff
@@ -94,3 +94,5 @@ export const useGetPublicIfoData = (ifo: Ifo): PublicIfoData => {
 
   return { ...finalState, currencyPriceInUSD }
 }
+
+export default useGetPublicIfoData
```

Comment:
line 57 ard export.
Can remove line 57 `export` or just use it
`import { useGetPublicIfoV3Data } from 'views/Ifos/hooks/v3/useGetPublicIfoData'`

Reply:
Done